components/ErrorResponse.yml

```diff
@@ -0,0 +1,33 @@
+openapi: 3.0.0
+info:
+  version: "0.0.3-unstable"
+  title: VC HTTP API
+  description: This is an Experimental Open API Specification for the [VC Data Model](https://www.w3.org/TR/vc-data-model/).
+  license:
+    name: W3C Software and Document License
+    url: http://www.w3.org/Consortium/Legal/copyright-software.
+  contact:
+    name: GitHub Source Code
+    url: https://github.com/w3c-ccg/vc-api
+paths:
+components:
+  schemas:
+    ErrorResponse:
```

Comment:
finally! real and potentially useful errors! <3

Reply:
If people like this, I'll apply it to all 400 error codes. This standard object also enables us to define standard error code responses more specifically in the spec.